tests/ShadowDOM/js/Document.js

```diff
@@ -230,14 +233,18 @@ htmlSuite('Document', function() {
 
     div.offsetHeight;
 
-    var list = document.querySelectorAll('div /deep/ aa');
```

Comment:
Hmm, I might be missing context. By "look duplicated" do you mean similar tests for Document.querySelectorAll vs Element.querySelectorAll?

`Document` has very unusual wrapping strategy (because the global `document` can't be patched to be the wrapped document node), so it's usually worth extra tests there.